### ISSUE TITLE
Pre-trained weights for Panoptic CMU

### ISSUE THREAD

**Issue by nicolasugrinovic** [zju3dv/SMAP#44]
Thanks for the release of your excellent work, could you please provide the model's weights trained with the Panoptic CMU dataset as the first table in your paper presents results with this dataset?

Thanks in advance

**Comment by ZXin0305** [comment]
> 

hello, have you retrained the model on CMU dataset, and what's the result of the model??
The result after training it on the dataset is so bad by myself.

**Comment by nicolasugrinovic** [comment]
@ZXin0305  How many GPUs did you use for training?

**Comment by nicolasugrinovic** [comment]
@[raypine](https://github.com/raypine) could you provide the weights trained with Panoptic dataset?

**Comment by raypine** [comment]
Sorry for that. It is not preserved.
However, according to our experience, for Panoptic, to achieve the performance in our paper (even better) is easy, since the distribution of training data is close to that of testing data to some extent, e.g. the camera viewing angle, the environment.
If you want to perform the experiments on the Panoptic dataset, you could finetune our provided model or train from scratch on it.

**Comment by nicolasugrinovic** [comment]
Could you provide more detail about the training time? How many GPUs did you use and how long did it take you to train?
Thanks

**Comment by ZXin0305** [comment]
> 

Hello,i trained the model on CMU using 3 Nvidia Titan xp cards,and using 200k images.For a complete training process,it takes about half a month....Maybe the computation power is much lower than A100,the biggest batch size is only 6,so the training procedure is very slow on myself host sever.What is your situation？

**Comment by raypine** [comment]
The training epochs we set is relatively large and you can stop early according to the loss curve and performance on validation set you choose.

**Comment by raypine** [comment]
We use 8 1080Ti and the whole training takes several days (cannot remember, but not much due to the early stopping)
The time-consuming part is the label generation process since we should generate supervisions for all immediate representations.

**Comment by JasonChen-Coder** [comment]
Have you got pretrained weights on Panoptic?If do,could you please help me？I would appreciate it very much.
